docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -197,7 +197,26 @@ implementation is used:
   - Set to `false` to disable in-memory caching of metadata files on the 
     coordinator. This cache is not used when `fs.cache.enabled` is set to true.
   - `true`
-
+* - `iceberg.expire_snapshots.min-retention`
+  -  Minimal retention period for `expire_snapshot` procedure. Equivalent
+     session property is `expire_snapshots_min_retention`.
+  -  `7 DAYS` 
```

Comment:
We should use airlift duration syntax `7d` here. Same for the following `iceberg.remove_orphan_files.min-retention` config property.

Reply:
oh .. I think thats invalid as @ebyhr also mentions. Should be https://trino.io/docs/current/admin/properties.html#duration and link to